ExtraDry/ExtraDry.Blazor/Components/Standard/Suspense.razor.cs

```diff
@@ -55,6 +55,12 @@ public partial class Suspense<TModel> : ComponentBase, IExtraDryComponent {
     [Parameter]
     public IndicatorSize Size { get; set; } = IndicatorSize.Standard;
 
+    /// <summary>
+    /// Which Indicator Icons should be displayed. Defaults to All
+    /// </summary>
+    [Parameter]
+    public IndicatorIcon ShowIcons { get; set; } = IndicatorIcon.All;
```

Comment:
Did a quick search of ED and the property `bool ShowIcon` is used in a few locations.  These seems like it would confuse a developer by having to remember different but similarly named properties.  Can this one be just a bool?  Is there a use-case where some icons are shown and others are not?